osh.spec

```diff
@@ -107,17 +107,23 @@ Obsoletes: covscan-hub < %{version}
 %description hub
 OpenScanHub xml-rpc interface and web application
 
-# define osh-{worker,hub}-conf-devel subpackages
-%(for sub in worker hub; do
-cat << EOF
-%package ${sub}-conf-devel
-Summary: OpenScanHub ${sub} devel configuration
-Provides: osh-${sub}-conf = %{version}-%{release}
-Conflicts: osh-${sub}-conf
-%description ${sub}-conf-devel
-OpenScanHub ${sub} devel configuration
-EOF
-done)
+%package hub-conf-devel
+Summary: OpenScanHub hub devel configuration
+Provides: osh-hub-conf = %{version}-%{release}
+Conflicts: osh-hub-conf
+Requires: httpd-filesystem
+Requires: osh-hub
```

Comment:
We should explain in the commit message why we are adding the dependency.  I do not think it is obvious without the context.

Reply:
Addressed in 4be7656 and db72d39.